Package installation requires cmake binary

**Checklist**

<!-- Mark with [x] steps you performed before submitting the issue -->

- [ x] I have read [Caveats](https://github.com/m-radzikowski/aws-sdk-client-mock#caveats) documentation and didn't find a solution for this problem there.

**Bug description**

A dependency requires aws-crt, which in turn requires cmake-js, which requires a local cmake installation. Which together makes this package harder to sell to the rest of the team.

Could you please list this requirement in the Install doc?

Relevant output from `npm install -D aws-sdk-client-mock -ddd` (`/path/to/project` replaces sensitive information):

```sh
npm ERR! path /path/to/project/node_modules/aws-crt
npm ERR! command failed
npm ERR! command sh -c node ./scripts/install.js
npm ERR! info TOOL Using Unix Makefiles generator.
npm ERR! info TOOL Building only the install target, as specified from the command line.
npm ERR! ERR! OMG CMake executable is not found. Please use your system's package manager to install it, or you can get installers from there: http://cmake.org.
npm ERR! /path/to/project/node_modules/cmake-js/lib/cMake.js:117
npm ERR!         throw new Error("CMake executable is not found. Please use your system's package manager to install it, or you can get installers from there: http://cmake.org.");
npm ERR!               ^
npm ERR!
npm ERR! Error: CMake executable is not found. Please use your system's package manager to install it, or you can get installers from there: http://cmake.org.
npm ERR!     at CMake.verifyIfAvailable (/path/to/project/node_modules/cmake-js/lib/cMake.js:117:15)
npm ERR!     at CMake.build (/path/to/project/node_modules/cmake-js/lib/cMake.js:287:10)
npm ERR!     at BuildSystem._invokeCMake (/path/to/project/node_modules/cmake-js/lib/buildSystem.js:70:40)
npm ERR!     at async buildFromRemoteSource (/path/to/project/node_modules/aws-crt/scripts/build.js:148:5)
```

**Environment**

- Node version: v16.13.0
- Testing lib and version: jest@27.4.3
- Typescript version: 4.4.3
- AWS SDK v3 Client mock version: 0.5.6, presumably
- AWS JS SDK libs and versions:
  - @aws-sdk/client-sqs: 3.43.0
- Platform: macOS 12.1, Apple M1 Pro chipset

--- a/README.md
+++ b/README.md
@@ -74,14 +74,19 @@ and define returned results depending on the `Command` type and payload.
 ### Install
 
 ```bash
-npm install -D aws-sdk-client-mock
+yarn add -D aws-sdk-client-mock
 ```
 
 or
 
 ```bash
-yarn add -D aws-sdk-client-mock
+npm install -D aws-sdk-client-mock
 ```
+
+With `npm`, peer dependencies will be installed, including `aws-crt`.
+**It requires `cmake` installed locally.**
+You can install it with `brew install cmake` or similar.
+See full explanation [here](https://github.com/m-radzikowski/aws-sdk-client-mock/issues/69#issuecomment-1035531094).
 
 ### Import
 

